tools/stress-cluster/cluster/kubernetes/stress-test-addons/file-plugin/plugin.yaml

```diff
@@ -0,0 +1,7 @@
+name: file
+version: 0.1
+description: Enables file:// protocol for helm repositories
```

Comment:
```suggestion
description: Enables local file protocol for helm repositories
```

Reply:
In this case I want to specifically reference that this handles the file URI scheme (https://en.wikipedia.org/wiki/File_URI_scheme).